www/nginx/Makefile

```diff
@@ -1,5 +1,6 @@
 PLUGIN_NAME=		nginx
 PLUGIN_VERSION=		1.24
+PLUGIN_REVISION=  1
```

Comment:
```suggestion
```

:)

Reply:
reverted to major version. conflict resolved)